source/docs/software/vscode-overview/3rd-party-libraries.rst

```diff
@@ -1,33 +1,38 @@
 3rd Party Libraries
 ===================
 
-Teams that are not using parts provided in the KoP will likely need to install external dependencies. Please view the below section to see information on adding an external library.
+Teams that are using non-PWM motor controllers or advanced sensors will most likely need to install external vendor dependencies. Please view the below section to see information on adding an external library.
 
-The Mechanism
--------------
+What are vendor dependencies?
+-----------------------------
 
-In support of this effort NI (for LabVIEW) and FIRST/WPI (for Java/C++) have developed mechanisms that should make it easy for vendors to plug their code into the WPILib software and for teams to use that code once it has been installed. A brief description of how the system works for each language can be found below.
+A vendor dependency is a way for vendors like CTRE, and REV to add their own custom code that interfaces with their motor controllers and other devices. This way you can interact with your devices via CAN and have access to more complex and in-depth features than traditional PWM control.
```

Comment:
```suggestion
A vendor dependency is a way for vendors like CTRE, REV, and others to add their own custom code that interfaces with their motor controllers and other devices. This way you can interact with your devices via CAN and have access to more complex and in-depth features than traditional PWM control.
```

Reply:
Avoid verbiage such as "like". Replace with "such-ass" or parenthesis. Additionally, avoid addressing the reader directly with "you". Instead refer to the reader as "the user" or "the team".